src/test/groovy/org/prebid/server/functional/tests/SchainSpec.groovy

```diff
@@ -154,4 +156,40 @@ class SchainSpec extends BaseSpec {
         def bidderRequest = bidder.getBidderRequest(bidRequest.id)
         assert bidderRequest.source?.schain?.nodes == [GLOBAL_SUPPLY_SCHAIN_NODE]
     }
+
+    def "PBS should use source.ext.schain when ext.prebid.schains.bidder doesn't requested"() {
```

Comment:
`PBS should use source.ext.schain when ext.prebid.schains.bidder isn't requested`